### DISCUSSION TITLE
The build time varies significantly each time

### DISCUSSION THREAD

**Discussion by cjnoname** [web-infra-dev/rspack#7923] [Q&A]
Hi,

Rspack is not performing consistently in my applications. Sometimes it takes 3-5 seconds to build, which is great. However, other times it takes 26 seconds, and in the worst case, it can take 56 seconds. I’m not sure why this happens.

Typically, the first build after pulling new code takes longer, but there doesn't seem to be a consistent pattern.

Anyone can help to explain why? Normally it stucks at "asset" steps.

**Comment by hardfist** [comment]
maybe related to Antivirus software which may take sometime to check whether asset is virus

**Reply by cjnoname** [reply]
Mate, I am using Mac and there's no anti virus software installed on my Mac

**Reply by cjnoname** [reply]
> maybe related to Antivirus software which may take sometime to check whether asset is virus

It stucks in this step: "sealing asset processing"

**Reply by AsuraKev** [reply]
I think I see this sometimes, at first i thought its a cool start thing but the build time actually fluctuates

**Reply by cjnoname** [reply]
> I think I see this sometimes, at first i thought its a cool start thing but the build time actually fluctuates

especially with fork-ts-checker-webpack-plugin and eslint plugin enabled.

**Reply by cjnoname** [reply]
> maybe related to Antivirus software which may take sometime to check whether asset is virus

Could you please provide an official reason for this? Is this normal behavior due to certain factors?
